types/order.ts

```diff
@@ -6,3 +6,13 @@ export interface Order {
   insurance: RadioOption | null;
   license: File | null;
 }
+
+export interface ProfileOrder {
+  id: string;
+  title?: string | null;
+  image?: string | null;
+  price?: number | null;
+  date?: string | null;
+  status?: string;
+  insurance?: string | null;
+}
```

Comment:
This is not implemented yet in Supabase right? Just a mock data and mock types?

Reply:
Yep, it's mock. Not implemented yet